deployments/rialto/update.sh

```diff
@@ -1,24 +1,40 @@
 #!/bin/bash
 
 # Script used to periodically update the network.
+#
+# If used with no flags then it updates the network using Docker
+# images from the Docker Hub. If used with the `-g` a.k.a the Git
+# flag then it'll build the Docker containers from GitHub.
 
 set -xeu
 
-git pull
+git_flag=false
 
 # Update *_BRIDGE_HASH in .env
 function update_hash {
 	sed -i "/$1/d" .env || true
 	echo "$1=$(git rev-parse HEAD)" >> .env
 }
 
-case "${1:-all}" in
-	all) update_hash "NODE_BRIDGE_HASH"; update_hash "ETH_BRIDGE_HASH"; update_hash "RELAY_BRIDGE_HASH";;
-	node) update_hash "NODE_BRIDGE_HASH";;
-	relay) update_hash "RELAY_BRIDGE_HASH";;
-	eth) update_hash "ETH_BRIDGE_HASH";;
-	*) echo "Invalid parameter: $1 (expected all/node/relay/eth)"; exit 1;;
-esac
+while getopts 'g:' flag; do
```

Comment:
I think we can avoid `getops` and simply add another branch to `case`, no?

Reply:
Actually do we really need a flag? Maybe let's control that with `.env` file, like we do with `WITH_PROXY` and other hashes? The reason for that would be that you most likely want the same behaviour on every run (i.e. always build git or from images).

Reply:
Yeah I can give that a try. I think it sounds reasonable